experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -185,10 +186,22 @@ func checkEntryPubKeyMatchesExpectedKey(rekordEntry *rekord.V001Entry, prodTeamK
 	return nil
 }
 
+// compareEndorsementAndRekordHash compares the hash referenced in a
+// rekord entry to the hash of input bytes.  For this use-case, the input
+// bytes will be an endorsement file
+func compareEndorsementAndRekordHash(rekordEntry *rekord.V001Entry, endorsementBytes []byte) error {
+	endorsementHash := fmt.Sprintf("%x", sha256.Sum256(endorsementBytes))
+	if endorsementHash != *rekordEntry.RekordObj.Data.Hash.Value {
+		return fmt.Errorf("Hash values of endorsement bytes and rekor entry not equal. endorsementHash: %s, rekorHash: %v",
+			endorsementHash, *rekordEntry.RekordObj.Data.Hash.Value)
+	}
+	return nil
+}
+
 // VerifyRekorEntry verifies a rekord entry by checking that the signature
 // it includes is valid, that the inclusion proof is valid, and that it
 // was created using a public key for the product team that we trust.
-func VerifyRekorEntry(rekorLogEntryBytes []byte, productTeamKeyBytes []byte) error {
+func VerifyRekorEntry(rekorLogEntryBytes []byte, productTeamKeyBytes []byte, endorsementFilePath string) error {
```

Comment:
I'd suggest passing the endorsement bytes to this function to be consistent with rest of the arguments.

Reply:
Fixed